vmdb/app/models/foreman_provision_task.rb

```diff
@@ -0,0 +1,66 @@
+require 'manageiq_foreman'
+class ForemanProvisionTask < MiqRequestTask
+  # provisioning builds the box
+  # vs alternative that just sets the attributes and doesn't rebuild
+  def build?
```

Comment:
The state machine methods are typically in a subdirectory based on the class name in a file called state_machine.rb and this file would ```include_concern 'StateMachine'```.